user.py

```diff
@@ -0,0 +1,6 @@
+import random
+class User:
+    def __init__(self, name, username):
+        self.name = name
+        self.username = username
+        self.userId = random.randint(1, 1000) * random.randint(1, 50)
```

Comment:
@hannahhall Check out this library.

https://docs.python.org/3.3/library/uuid.html